apis/v1alpha1/installationTypesHelper_test.go

```diff
@@ -128,14 +101,9 @@ func TestManagedInstallType(t *testing.T) {
 			installType:     InstallationTypeMultitenantManagedApi,
 			expectedOutcome: true,
 		},
-		{
-			name:            "test that installation is managed",
-			installType:     InstallationTypeManaged,
-			expectedOutcome: true,
-		},
 		{
 			name:            "test that installation is not managed",
-			installType:     InstallationTypeSelfManaged,
+			installType:     "Dummy Type",
```

Comment:
since we are only leaving the multitenant and managed-api I'm not sure what is the benefit in testing if a type is not managed since it can't be anything else

Reply:
Currently it can't be anything else. Unit test help during development and future refactors. It is for future refactor that this test case is addressing. As there is no maintenance cost to having this test case, I don't see the need to remove it.

Reply:
On second pass the function under test was removable. This unit test has being fully removed.

Reply:
Thanks